### DISCUSSION TITLE
Bad user experience with Generate styles feature

### DISCUSSION THREAD

**Discussion by oscklm** [rsayfutdinov/Pikka#47] [Ideas]
Hello,

I just bought the tool, but sadly it does not live up to what I had expected and, it is especially frustrating to use the generate styles feature, which was my key interest and reason for buying.

### My workflow
I have a reference png with some colours, and I want to use the generate styles to make some nice tints/shades from the colors.

1. I open the generate styles
2. I then color pick the color I wanna use from the png I have open on my desktop
3. I then manually plug in the values into the generate styles window and pick shades or tint and then set it to linear

### Issue
When picking a color, the "Generate styles" window disappears, and when you then open it from the menu bar, a new window opens and the old "Generate styles" window opens behind all other windows.

### Feature request
1. A way to pick a color and have it be passed into the "Generate styles" window automatically if u have one open
2. Have the "Generate styles" window remember your last settings, ex. whether you picked shades/tint and what type of curve u chose 

I hope this feedback can help, I would love to see an improvement on this. If not possible, I will be issuing a refund request, as the tool sadly then does not deliver in terms of my expectations and needs.

**Comment by rsayfutdinov** [comment]
Hey Oscar,
Thank you for your feedback. I will make the improvements within the next update.
